lean4-infoview-api/src/infoviewApi.ts

```diff
@@ -140,6 +140,13 @@ export interface InfoviewApi {
    */
   requestedAction(action: InfoviewAction): Promise<void>;
 
+  /**
+   * Must fire whenever the user requests to go to a definition using the infoview context menu.
+   * `interactiveCodeTagId` is obtainable from the `data-vscode-context` field in
```

Comment:
Could we point to the VSCode context menu docs here? It's probably not really possible to understand what is going on here without that context.

Reply:
Sure!